### DISCUSSION TITLE
Pages manager problem

### DISCUSSION THREAD

**Discussion by bovealexandre** [GrapesJS/grapesjs#4429] [Q&A]
Hi,

I'm trying to use GrapesJS but I'm having a hard time with the PageManager. I wanted to know if it was possible to keep the head between pages because I'm using Tailwind with my own config. so what are the options on the select function? and why does the add function only works once per refresh?

**Comment by bovealexandre** [comment]
ok I did the code for tailwind with my own config there is the whole scripts tags in the new canvas on the head tags but it doesn't apply to the page and when I refresh the canvas it deletes the children head tag but I still can't understand why I can't add more than one page

**Reply by artf** [reply]
Hi @bovealexandre are you able to provide a reproducible demo of your case?
